### PR TITLE
alpha and beta divesity section

### PR DESCRIPTION
**Pull request by shigdel**
Here i have updated according to the Leo`s suggestion- making different section for alpha and diversity

### CODE REVIEW

File: beta-diversity.rmd [hunk 1]
```diff
@@ -0,0 +1,338 @@
+---
+title: "Beta-diversity"
+output: html_document
+---
+
```
Comment (ChouaibB) [inline beta-diversity.rmd:5]: Could you add here:
```{r setup, echo=FALSE, results="asis"}
library(rebook)
chapterPreamble()
```

**Review by ChouaibB — CHANGES_REQUESTED** [review]

File: beta-diversity.rmd (hunk 1 above)
Comment (ChouaibB) [reply beta-diversity.rmd:5]: How about the "_bookdown.yml"?

Comment (shigdel) [reply beta-diversity.rmd:5]: i have edited on the same file it should be there

File: alpha-diversity.rmd [hunk 2]
```diff
@@ -0,0 +1,154 @@
+---
+title: "alpha-diversity.rmd"
```
Comment (antagomir) [inline alpha-diversity.rmd:2]: Let's not use dash in file names, and let us unify file naming, we have used "Rmd" elsewhere.

Change the file names "alpha-diversity.rmd" -> "alpha_diversity.Rmd" and same for other Rmd files.

**Review by antagomir — CHANGES_REQUESTED** [review]
I do not see changes in _bookdown.yml file?

File: beta-diversity.rmd (hunk 1 above)
Comment (antagomir) [reply beta-diversity.rmd:5]: The "Files changed" tab above does not show any changes in "_bookdown.yml"? Have you committed the changes?

Comment (shigdel) [reply beta-diversity.rmd:5]: fixes have been made at branch issue_34 #34

File: alpha-diversity.rmd (hunk 2 above)
Comment (antagomir) [reply alpha-diversity.rmd:2]: Is this done now?

**Comment by antagomir** [comment]
Let us close this after all feedback has been resolved.

**Comment by antagomir** [comment]
Ok the branch seems to have been deleted but changes are still necessary.

**Comment by shigdel** [comment]
these were duplicate branches (mybranch and myBranch). corresponding changes and fixes have been made at issue_34 #34

**Pull request closed without merging**